README.md

```diff
@@ -1138,6 +1138,10 @@ The `jumpstation` connection options support the same values (for as much as it
 
 <a name="release_history"></a>
 # Release History
+* Overthere 5.0.21 (07-Oct-2020)
+    * Make artifact-copy-strategy work on satellite
```

Comment:
No mention of smbj upgrade ?

Reply:
Previous review cmts: Version changes are cumulative. That means we don't have to mention " and Make artifact-copy-strategy work on satellite" on this line. 
Just removed version .

Reply:
There is no mention of SMBJ upgrade ?